providers/redis/pyproject.toml

```
@@ -62,11 +62,19 @@ dependencies = [
     "redis>=4.5.2,!=4.5.5,!=5.0.2",
 ]
 
+# The optional dependencies should be modified in place in the generated file
+# Any change in the dependencies is preserved when the file is regenerated
+[project.optional-dependencies]
+"common.messaging" = [
+    "apache-airflow-providers-common-messaging"
```

```suggestion
    "apache-airflow-providers-common-messaging>1.0.3"
```

I think we have to change it to 1.0.3 as minimum because of #51774 - also in other providers. So it might be good idea to add it here. I will change the other ones.

---

PR here: https://github.com/apache/airflow/pull/54160

---

Sure! Thanks for pointing out!